supabase.types.ts

```diff
@@ -36,33 +36,72 @@ export interface Database {
     Tables: {
       landlord_private_profiles: {
         Row: {
+          city: string | null
+          country: string
+          county: string | null
+          house: string
           id: string
+          phone_number: string
+          postcode: string
+          street: string | null
         }
         Insert: {
+          city?: string | null
+          country: string
+          county?: string | null
+          house: string
           id: string
+          phone_number: string
+          postcode: string
+          street?: string | null
         }
         Update: {
+          city?: string | null
+          country?: string
+          county?: string | null
+          house?: string
           id?: string
+          phone_number?: string
+          postcode?: string
+          street?: string | null
         }
         Relationships: [
           {
             foreignKeyName: "landlord_private_profiles_id_fkey"
             columns: ["id"]
             isOneToOne: true
             referencedRelation: "user_profiles"
-            referencedColumns: ["id"]
+            referencedColumns: ["user_id"]
           }
         ]
       }
       landlord_public_profiles: {
         Row: {
+          bio: string | null
           id: string
+          profile_image_id: string | null
+          rating: number
```

Comment:
Not sure if we want to be storing the rating directly in the profile.

It would likely make more sense to calculate at request time with some caching as we are with functions like we do in the `get_properties_with_ratings` you can see in the '20240215131101_define_property_rating_functions' migration or in the studio.

Creating these functions is a separate task (RRS-64), but we probably don't need to include the field here

Up to you if you want to remove it now or we can deal with it when we handle that task next week.

Reply:
Good point, I can remove it now. Also I was looking at the naming of the id within the landlord tables, should we change it to user_id to be consistent with the reviewer tables and also to be clear that it is dependent on the id in the users table?

Reply:
That would make sense, we can do that